app/models/subnet.rb

```diff
@@ -58,6 +58,12 @@ def model_name
     :api_description => N_('HTTPBoot Proxy ID to use within this subnet'),
     :description => N_('HTTPBoot Proxy to use within this subnet')
 
+  belongs_to_proxy :externalipam,
+    :feature => N_('External IPAM'),
+    :label => N_('IP Address Management with External IPAM'),
```

Comment:
Please keep the label just `External IPAM` it breaks the page (too long of a text).